Diagnostics/HealthChecker/DataCollection/ServerInformation/Get-OperatingSystemInformation.ps1

```diff
@@ -96,7 +96,7 @@ Function Get-OperatingSystemInformation {
         -ScriptBlockDescription "Getting Current Time Zone" `
         -CatchActionFunction ${Function:Invoke-CatchActions}
     $osInformation.TLSSettings = Get-AllTlsSettingsFromRegistry -MachineName $Script:Server -CatchActionFunction ${Function:Invoke-CatchActions}
-    $osInformation.VcRedistributable = Get-VisualCRedistributableInstalledVersion -ComputerName $Script:Server -CatchActionFunction ${Function:Invoke-CatchActions}
+    $osInformation.VcRedistributable = @(Get-VisualCRedistributableInstalledVersion -ComputerName $Script:Server -CatchActionFunction ${Function:Invoke-CatchActions})
```

Comment:
Might be a problem for HC because we check if it's `$null` before looping through the `Test-VisualCRedistributableUpToDate` tests. Still validating. Please do not merge yet.

Reply:
Ok, the change causes that the following condition always applies `if ($null -ne $osInformation.VcRedistributable) { }` and so we're looping through the `Test-VisualCRedistributableUpToDate` cases. Result is the same (version displayed as `Unknown`) but we can avoid this not necessary work. I recommend reverting the change made in `Get-OperatingSystemInformation`

Reply:
Done.